libs/sqlite3/Makefile

```diff
@@ -94,12 +94,32 @@ define Package/sqlite3-cli/config
   source "$(SOURCE)/Config-cli.in"
 endef
 
+ifeq ($(CONFIG_SQLITE3_LINENOISE),y)
+LINENOISE_COMMIT:=880b94130ffa5f8236392392b447ff2234b11983
+define Download/linenoise
+	PROTO:=git
+	URL:=https://github.com/antirez/linenoise.git
```

Comment:
Is it possible to make `linenoise` as a separated package?

Reply:
AFAIK, linenoise supports embedding only. At least its [Makefile](https://github.com/antirez/linenoise/blob/880b94130ffa5f8236392392b447ff2234b11983/Makefile) doesn't provide an option to build it as a shared library.

Reply:
Okay thanks for clarification.